libbeat/scripts/Makefile

```diff
@@ -305,16 +306,8 @@ index-template: ## @build Generate index templates for the given $VERSION. This
 ### KIBANA FILES HANDLING ###
 ES_URL?=http://localhost:9200
 
-.PHONY: export-dashboards
-export-dashboards: python-env update
-	. ${PYTHON_ENV}/bin/activate && python ${ES_BEATS}/dev-tools/export_dashboards.py --url ${ES_URL} --dir $(shell pwd)/_meta/kibana --regex ${BEAT_NAME}-*
```

Comment:
Could we get `export-dashboards` and `import-dashboards` targets based on the go code? I'm good with follow up PR for them.